src/content/self-hosted/administration/configuration.mdx

```diff
@@ -37,23 +37,39 @@ Want to use Okteto with a bigger team? [Let's talk](https://okteto.com/#talktous
 
 ### subdomain
 
-The domain (or subdomain) managed by Okteto.
-
-Your Okteto instance will be available at `okteto.$SUBDOMAIN`. All ingresses created by okteto will use it as well (e.g. https://app-$NAMESPACE.$SUBDOMAIN)
+Okteto offers a wildcard subdomain feature that allows you to expose secure public endpoints for your development environments.
+For instance, if you have a development environment named `app` in the `cindy` namespace, it will be accessible at https://app-cindy.example.com if `subdomain` is example.com.
```

Comment:
```suggestion
For example, if you have a development environment named `app` in the `cindy` namespace, it will be accessible at https://app-cindy.example.com if `subdomain` is example.com.
```